apps/src/studio/levels.js

```diff
@@ -1951,23 +1932,19 @@ levels.js_hoc2015_event_choose_character =
     "function whenDown() {",
     "  moveDown();",
     "}"].join("\n"),
-
   "sortDrawOrder": true,
   "wallMapCollisions": true,
   "blockMovingIntoWalls": true,
-  "gridAlignedMovement": false,
   "itemGridAlignedMovement": true,
   "removeItemsWhenActorCollides": true,
-  "markerHeight": 50,
-  "markerWidth": 50,
   "delayCompletion": 2000,
   "floatingScore": true,
   "edgeCollisions": "true",
-
   "map": [[0, 0, 0, 0, 0, 0, 0, 0], [0, 0, 0, 0, 0, 0, 0, 0], [0, 0, 0, 0, 0, 0, 0, 0], [0, 0, 0, 0, 0, 0, 0, 0], [0, 0, 0, 16, 0, 0, 0, 0], [0, 0, 0, 0, 0, 0, 0, 0], [0, 0, 0, 0, 0, 0, 0, 0], [0, 0, 0, 0, 0, 0, 0, 0]],
-
   "instructions": "\"Time to visit another planet.\"",
   "instructions2": "Use the dropdown to change the background.  Now find a command to change your BOT.",
+  "timeoutFailureTick": 600,
+  "timeoutAfterWhenRun": true,
```

Comment:
Yeah, though note I believe this is a no-op if you have any event based code (i.e. if I have a chunk of blocks underneath a when click)